contracts/utils/BatchTransfer.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+contract BatchTransfer is Ownable {
```

Comment:
for futher gas optimization and usage,
we won't be sending the tokens from msg.sender, but instead from the contract:

My intention is to mint directly to the contract address and simply do a transfer
without need of allowing.